src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -146,6 +146,8 @@ public static void validateDictionaries( final String name1,
                                              final SAMSequenceDictionary dict2,
                                              final boolean requireSuperset,
                                              final boolean checkContigOrdering ) {
+        Utils.nonNull(dict1, "Something went wrong with sequence dictionary detection, check that "+name1+" has a valid sequence dictionary");
+        Utils.nonNull(dict2, "Something went wrong with sequence dictionary detection, check that "+name2+" has a valid sequence dictionary");
```

Comment:
Can you add a test that fails without this patch, and passes with it?

Reply:
done